includes/Plugin.php

```diff
@@ -213,6 +217,12 @@ public function register() {
 
 		$activation_notice = new Activation_Notice( $activation_flag );
 		$activation_notice->init();
+
+		$this->integrations['site-kit'] = new Site_Kit( $this->analytics );
+		add_action( 'init', [ $this->integrations['site-kit'], 'init' ] );
```

Comment:
```suggestion
		$site-kit = new Site_Kit( $this->analytics );
		add_action( 'init', [ $site-kit, 'init' ] );
		$this->integrations['site-kit'] = $site-kit;
```
Would this fix the issue?

Reply:
Hmm that works